index.html

```diff
@@ -0,0 +1,163 @@
+      </head>
+      <body>
+        <div id="page-wrapper">
+          <header id="header">
+            <div class="logo">
+              <img
+                id="header-img"
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+                alt="original trombones logo"
+              />
+            </div>
+
+            <nav id="nav-bar">
+              <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li>
+                  <a class="nav-link" href="#how-it-works">How It Works</a>
+                </li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+              </ul>
+            </nav>
+          </header>
+
+          <div class="container"></div>
+
+          <section id="hero">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+              <input
+                name="email"
+                id="email"
+                type="email"
+                placeholder="Enter your email address"
+                required
+              />
+              <input
+                id="submit"
+                type="submit"
+                value="Get Started"
+                class="btn"
+              />
+            </form>
+          </section>
+
+          <div class="container">
+            <section id="features">
+              <div class="grid">
+                <div class="icon"><i class="fa fa-3x fa-fire"></i></div>
+                <div class="desc">
+                  <h2>Premium Materials</h2>
+                  <p>
+                    Our trombones use the shiniest brass which is sourced
+                    locally. This will increase the longevity of your purchase.
+                  </p>
+                </div>
+              </div>
+              <div class="grid">
+                <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+                <div class="desc">
+                  <h2>Fast Shipping</h2>
+                  <p>
+                    We make sure you recieve your trombone as soon as we have
+                    finished making it. We also provide free returns if you are
+                    not satisfied.
+                  </p>
+                </div>
+              </div>
+              <div class="grid">
+                <div class="icon">
+                  <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+                </div>
+                <div class="desc">
+                  <h2>Quality Assurance</h2>
+                  <p>
+                    For every purchase you make, we will ensure there are no
+                    damages or faults and we will check and test the pitch of
+                    your instrument.
+                  </p>
+                </div>
+              </div>
+            </section>
+          </div>
+        </div>
```

Comment:
Please correct the width you added to this section, it is causing a horizontal scroll

Reply:
- [ ] You have done well to use the polite language in communication. Tough we should make sure that the explaination is detailed. For example, you can explain the dangers of horizontal scroll and why its needed to be fixed.
- [ ] [OPTIONAL] In addition, please consider using the imojis and gifs. This will make you review comfortable to the student and more friendly.